code/XMLTester.xml

```diff
@@ -2536,169 +2536,213 @@
+                                        <address>192.168.1.113</address>
+                                        <port>64863</port>
+                                    </udpv4>
+                                </locator>
+                            </metatrafficUnicastLocatorList>
+                        </RemoteServer>
+                    </discoveryServersList>
+                </discovery_config>
+            </builtin>
+        </rtps>
+    </participant>
 <!--><-->
 
 <!-->CONF-SERVER-DISCOVERY-PROTOCOL<-->
-<participant profile_name="participant_discovery_protocol_alt" >
-  <rtps>
-    <builtin>
-        <discovery_config>
-          <discoveryProtocol>CLIENT</discoveryProtocol>
-          <!-- alternatives
-          <discoveryProtocol>SERVER</discoveryProtocol>
-          <discoveryProtocol>BACKUP</discoveryProtocol>
-          -->
-        </discovery_config>
-    </builtin>
-  </rtps>
-</participant>
+<!--
+<?xml version="1.0" encoding="UTF-8" ?>
+<profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+-->
+    <participant profile_name="participant_discovery_protocol_alt" >
+        <rtps>
+            <builtin>
+                <discovery_config>
+                    <discoveryProtocol>CLIENT</discoveryProtocol>
+                    <!-- alternatives
+                    <discoveryProtocol>SERVER</discoveryProtocol>
+                    <discoveryProtocol>BACKUP</discoveryProtocol>
+                    -->
+                    </discovery_config>
+            </builtin>
+        </rtps>
+    </participant>
 <!--><-->
 
 <!-->CONF-SERVER-CLIENT-PING<-->
-<participant profile_name="participant_profile_client_ping" >
-  <rtps>
-    <builtin>
-        <discovery_config>
-          <clientAnnouncementPeriod>
-            <!-- change default to 250 ms -->
-            <nanosec>250000000</nanosec>
-          </clientAnnouncementPeriod>
-        </discovery_config>
-    </builtin>
-  </rtps>
-</participant>
+<!--
+<?xml version="1.0" encoding="UTF-8" ?>
+<profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+-->
+    <participant profile_name="participant_profile_ping" >
+        <rtps>
+            <builtin>
+                <discovery_config>
+                    <clientAnnouncementPeriod>
+                        <!-- change default to 250 ms -->
+                        <nanosec>250000000</nanosec>
+                    </clientAnnouncementPeriod>
+                </discovery_config>
+            </builtin>
+        </rtps>
+    </participant>
 <!--><-->
 
-<!-->CONF-SERVER-SERVER-PING<-->
-<participant profile_name="participant_profile_server_ping" >
-  <rtps>
-    <builtin>
-        <discovery_config>
-          <clientAnnouncementPeriod>
-            <!-- change default to 250 ms -->
-            <nanosec>250000000</nanosec>
-          </clientAnnouncementPeriod>
-        </discovery_config>
-    </builtin>
-  </rtps>
-</participant>
+<!-->CONF_SERVER_FULL_EXAMPLE<-->
```

Comment:
I would also add the case of the server pinging other servers. It is mentioned in the text and it should be also in the full example case.